CHANGELOG.md

```diff
@@ -1,5 +1,7 @@
 # Change Log
 
+- [filesystem] add text input and nagivate up icon to file dialog [#8748](https://github.com/eclipse-theia/theia/pull/8748)
```

Comment:
nit: it should be placed under a new `## v1.10.0` entry, but I can always update it when preparing the changelog for the upcoming release.